Header header
uint32 sentence_id
uint32 stream_id
uint32[] frame_ids
time start
time end
uint64 duration
string speaker
string sentence